### ISSUE TITLE
kafka service should wait for zookeeper

### ISSUE THREAD

**Issue by igalic** [voxpupuli/puppet-kafka#81]
When building a new server with kafka (and zookeeper) we see that kafka doesn't come up until after restarting zookeeper — and then starting kafka.

We'd expect new servers to simply provision and have a running and functioning kafka broker.

Perhaps as a first step towards improvement we should specify ~depends~ in the systemd unit / systemv init script.?

**Comment by bastelfreak** [comment]
as an example kafka.service:
```ini
[Unit]
After=zookeeper.service
....
```

this will start zookeeper when you start kafka. zookeeper has to be enabled already.

---

However, if you are unsure if zookeeper is already enabled you can use the following to start it  independently from his state while starting kafka:
```ini
[Unit]
Wants=zookeeper.service
After=zookeeper.service
....
```

---

last option: maybe you want to auto enable zookeeper while enabling kafka:
```ini
[Unit]
Wants=zookeeper.service
After=zookeeper.service
Requires=zookeeper.service
....
```


_(this is all untested)_

**Comment by igalic** [comment]
this was fixed in #82
